cmd/cfgfiles/tapir-pop.yaml

```
@@ -0,0 +1,66 @@
+
+cli:
+    tapir-pop:
+        url: https://127.0.0.1:9099/api/v1
+        tlsurl: https://127.0.0.1:9098/api/v1
+        apikey: be-nice-to-a-bad-tempered-tapir
+
+apiserver:
+    active: true
+    name: TAPIR-POP API Server
+    key: be-nice-to-a-bad-tempered-tapir
+    addresses: [ 0.0.0.0:9099 ]
```

YAMLlint (1.37.1)

[warning] cannot infer indentation: unexpected token

(indentation)